src/Mapping/DefaultQuoteStrategy.php

```diff
@@ -52,7 +61,10 @@ public function getTableName(ClassMetadata $class, AbstractPlatform $platform):
     public function getSequenceName(array $definition, ClassMetadata $class, AbstractPlatform $platform): string
     {
         return isset($definition['quoted'])
-            ? $platform->quoteSingleIdentifier($definition['sequenceName'])
+            ? implode('.', array_map(
```

Comment:
I have little understanding of what tradeoffs are acceptable here, so I cannot provide a reasonable feedback.

Every `implode('.', $name)` is a potential bug, which is why `quoteIdentifier()` was deprecated in the first place.

Assuming

Reply:
I initially wrote a separate test just for the reported issue, and the other part of the patch fixed it. This is here to fix another issue found when I decided to reuse the existing test, if I recall correctly. In any case, I think that on a patch branch, doing this is the right thing, but that on the next minor, we might want deprecate relying on that implode. This will require further analysis as to the ramification of the deprecation though.